tests/Http/Three/SignupTest.php

```diff
@@ -113,10 +112,10 @@ public function testSignupsIndex()
     {
         factory(Signup::class, 10)->create();
 
-        $this->get('api/v3/signups');
+        $response = $this->get('api/v3/signups');
```

Comment:
how do you choose to use `get` vs `getJson`?

Reply:
Mmm, good catch. We should probably always use `getJson` since it sets the right `Accept` and `Content-Type` headers (although a lot of cases _do_ work without specifying those too).

Reply:
oh that is neat that it does that for you!